src/suites/cts/validation/create_bind_group_layout.spec.ts

```diff
@@ -0,0 +1,87 @@
+export const description = `
+Create bind group layout validation tests.
+`;
+
+import { TestGroup } from '../../../framework/index.js';
+
+import { ValidationTest } from './validation_test.js';
+
+function clone(descriptor: GPUBindGroupLayoutDescriptor): GPUBindGroupLayoutDescriptor {
+  return JSON.parse(JSON.stringify(descriptor));
+}
+
+export const g = new TestGroup(ValidationTest);
+
+g.test('some binding index was specified more than once', async t => {
+  const goodDescriptor: GPUBindGroupLayoutDescriptor = {
+    bindings: [
+      {
+        binding: 0,
+        visibility: GPUShaderStage.COMPUTE,
+        type: 'storage-buffer',
+      },
+      {
+        binding: 1,
+        visibility: GPUShaderStage.COMPUTE,
+        type: 'storage-buffer',
+      },
+    ],
+  };
+
+  const badDescriptor = clone(goodDescriptor);
+  badDescriptor.bindings![1].binding = 0;
+
+  // Control case
+  t.device.createBindGroupLayout(goodDescriptor);
+
+  // Binding index 0 can't be specified twice.
+  await t.expectValidationError(() => {
+    t.device.createBindGroupLayout(badDescriptor);
+  });
+});
+
+g.test('some binding index exceeds the maximum value', async t => {
```

Comment:
I think this may also change in spec, but we'll change the test when we get to it.

Reply:
Which issue should I link to for the TODO?

Reply:
Not that important, could do a TODO without issue or just leave it out. We'll easily find the test when we need to fix it.